cfgov/agreements/templates/agreements/issuer_select.html

```diff
@@ -1,5 +1,5 @@
 <select data-placeholder="Choose an issuer" class="chzn-select" tabindex="2" id="issuer_select">
 {% for i in issuers %}
-    <option value="{{ i.slug }}" {% ifequal selected i.slug %}selected{% endifequal %}>{{ i.name }}</option>
+    <option value="{{ i.slug }}" {% ifequal selected i.slug %}selected="selected"{% endifequal %}>{{ i.name }}</option>
```

Comment:
Why is this change required for this PR?

Reply:
This change goes against our HTML style guide.

Reply:
Because BeautifulSoup4 does not automatically populate "selected" as the value of the selected attribute, and BeautifulSoup3 is not Python3-compatible.

Reply:
Could that be addressed by modifying [this test](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/agreements/tests/test_legacy.py#L233)? It'd be nice to limit this PR to just enable Python3 testing vs. actually making tests work.

What exactly is the issue - is it that the tox environment setup fails if you try to install a requirement that isn't supported by Python 3?

Reply:
Yeah, modifying the test to simply check for the presence of the `selected` attribute would be preferable.

Reply:
I tried. Checking for `selected=""` does not match the expected `option`.

Reply:
this works for me:

```python
def get_selected_value(select_element):
    for item in select_element.find_all('option'):
        if item.get('selected', None) == '':
            return item.text
```

Reply:
@higs4281 Yeah, [this was the only alternative I could make work](https://github.com/cfpb/cfgov-refresh/pull/3793/commits/94d00b11e2fc9f32cea68634d92bf191e3dd0834#diff-8f93154e53c7a83b3dc584c02d90e339R233).

@chosak @Scotchester I just don't think a list comprehension to make bs4 match bs3's assumptions is the right way to go.

It's a minor thing — I think it's uglier code, but it's far more important to me to be able to run tests with Python 3. 😁

Reply:
DId you try `find_all('option', selected=True)`? See https://stackoverflow.com/a/39055066/279566

Reply:
@Scotchester I thought I had, but apparently I hadn't. Cheers! That works!